Gruntfile.coffee

```diff
@@ -68,7 +68,7 @@ module.exports = (grunt) ->
       coverage:
         command: "./node_modules/.bin/istanbul cover ./node_modules/.bin/jasmine-node --captureExceptions test && cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js && rm -rf ./coverage"
       jasmine:
-        command: "./node_modules/.bin/jasmine-node --verbose --captureExceptions test"
+        command: "./node_modules/.bin/jasmine-node --verbose --captureExceptions --coffee src/spec"
```

Comment:
It should be use the same latest version 

https://github.com/mhevery/jasmine-node/blob/master/package.json#L26
https://github.com/gruntjs/grunt-contrib-coffee/blob/master/package.json#L24